gemfire/spring-cloud-starter-stream-source-gemfire-cq/src/main/java/org/springframework/cloud/stream/app/gemfire/cq/source/GemfireCqSourceProperties.java

```diff
@@ -25,7 +25,7 @@
  *
  * @author David Turanski
  */
-@ConfigurationProperties
+@ConfigurationProperties("gemfire.cq")
```

Comment:
that makes sense - i was worried about collisions, but I think we are good here.